Mathlib/AlgebraicGeometry/AffineScheme.lean

```diff
@@ -712,6 +735,104 @@ lemma Scheme.eq_zeroLocus_of_isClosed_of_isAffine (X : Scheme.{u}) [IsAffine X]
 
 end ZeroLocus
 
+section Factorization
+
+variable {X : Scheme.{u}} {A : CommRingCat}
+
+/-- If `X ⟶ Spec A` is a morphism of schemes, then `Spec` of this ring is the scheme-theoretic
```

Comment:
```suggestion
/-- If `f : X ⟶ Spec A` is a morphism of schemes, then `Spec A` is the scheme-theoretic
```
Is this the intended meaning?

Reply:
Ah, "this ring" is the ring `specTargetImage f`, i.e. this refers to the ring defined in this definition.